### ISSUE TITLE
Ratio has zero denominator error from multicolumn bs

### ISSUE THREAD

**Issue by simonmichael** [simonmichael/hledger#535]
Related to #518:
```
$ hledger -f doc/hledger.journal bs -MA
Balance Sheet

hledger: Ratio has zero denominator
```

**Comment by mstksg** [comment]
Looks like it's related to the undesired behavior when a section has no entries.  Fixing that should fix both, it seems.  I'll try to take a crack at it.

**Comment by simonmichael** [comment]
Fixed by commit c8e4bc0 (which mistakenly referenced #518).
